website/docs/getting-started.md

```diff
@@ -3,10 +3,77 @@ id: getting-started
 title: Getting started
 ---
 
-:::note
+### Overview
 
-Ledokku is under active development.
+To get started with Ledokku you will have to set up your own server. If you already
+have experience with installing and running dokku, please skip the part below proceed directly to [**installation guide**](installation.md).
 
-🚀 Follow our [twitter](https://twitter.com/ledokku) to get notified once the project is ready.
+If you want to try out Ledokku and need guidance on how to set up your own server and install dokku on it, please follow these steps.
 
-:::
+- [Set up Digital Ocean account](#set-up-digital-ocean-account-and-get-100-credit)
+- [Create Docker Droplet](#create-docker-droplet)
+- [SSH into Digital Ocean Droplet](#ssh-into-digital-ocean-droplet)
+- [Install dokku on your Droplet](#install-dokku-on-your-droplet)
+- [Proceed to next page for further instructions](#proceed-to-next-page-to-install-ledokku)
+
+### Set up Digital Ocean account and get \$100 credit
+
+First you will need to open a Digital Ocean account and we recommend you do it with this [**link**](https://m.do.co/c/35f78321cb42) as it will it bring you **\$100 free credit** to play around for the first 60 days. More than enough to try out Ledokku.
+
+### Create Docker Droplet
+
+In Digital Ocean server's are referred as droplets and in our case we will
+be creating Docker Droplet to host our dokku apps on.
+
+1. Expand the **_Discover_** section on left side menu
+2. Click **_Marketplace_**
+3. Search for **_Docker_**
+
+!["Search for docker droplet"](/../static/img/searchDroplet.png)
```

Comment:
@Akirtovskis we should probably link the images that way https://v2.docusaurus.io/docs/static-assets#markdown-example